exchange/seat_non_bids.go

```diff
@@ -0,0 +1,68 @@
+package exchange
+
+import (
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/exchange/entities"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type seatNonBids struct {
+	seatNonBidsMap map[string][]openrtb_ext.NonBid
+}
+
+func newSeatNonBids() seatNonBids {
+	return seatNonBids{
+		seatNonBidsMap: make(map[string][]openrtb_ext.NonBid),
+	}
+}
+
+func (snb *seatNonBids) add(bid *entities.PbsOrtbBid, nonBidReason int, seat string) {
```

Comment:
Consider using a value receiver here instead of a pointer in receiver. In Go, it's good practice to always default to a value unless you need to set state or the structure is really large - neither which apply in this case.